src/store/traction/pacbio/runCreate/mutations.js

```diff
@@ -155,14 +155,21 @@ export default {
    * @param {Object} plates The plates for the run
    * @param {Object} wells The wells for the run
    * Adds the wells to state by plate number and well position, two dimensional array
+   * Adds the _destroy key to each well plate
    */
   populateWells: (state, { plates, wells }) => {
-    state.wells = splitDataByParent({
+    const wellsByPlate = splitDataByParent({
       data: wells,
       fn: dataToObjectByPosition,
       includeRelationships: true,
       parent: { parentData: plates, children: 'wells', key: 'plate_number' },
     })
+    // We need to add the _destroy key to each plate to handle well deletions
+    // eslint-disable-next-line no-unused-vars
+    Object.entries(wellsByPlate).forEach(([_plateNumber, plate]) => {
+      plate['_destroy'] = []
+    })
```

Comment:
yeah I did think about putting it there but I think i prefer it being defined upfront because if we only define it on deleteWell, then the _destroy flag may or may not exist, depending on if any wells are deleted. This makes the object less well defined.